### PR TITLE
Add a GameCube Backup Manager page.

### PR DESCRIPTION
**Pull request by thondorigithub1**
If anyone can find a GameCube Backup Manager alternative for mac/linux, I will include that.

### CODE REVIEW

File: _pages/en_US/nintendont.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+##### Getting Started
+
+There is no "guide" to use Nintendont. This is intended to help you learn how to use it, by giving you a quick start. You should be able to figure out all the great features of Nintendont by using it.
+{: .notice--info}
+
+###### Key Features
+
+1. You can use a GameCube controller to play your games.
+2. You can use a Wiimote to play your games, but you will need to use a Classic Controller Pro or a Wii U Pro Controller to use all of the game's features.
+3. You can use a USB keyboard to enter text in games that require it.
```
Comment (Snare-Hawk) [inline _pages/en_US/nintendont.md:42]: can we even call these key features? also, the use of "You can use" is repetitive

Comment (thondorigithub1) [reply _pages/en_US/nintendont.md:42]: nintendont file wasnt supposed to be there

**Comment by Snare-Hawk** [comment]
i can see why we didn't have a nintendont page to begin with, i think it'd need to be more fleshed out than what is there now
i'd happily merge the gamecube backup manager page though
i'll assign peter to review this, to give his opinion and we'll likely probably have to save a nintendont page for another PR
~~if this is the case, thonny don't delete the page just yet, i'll delete it right before i merge it teehee~~

**Comment by thondorigithub1** [comment]
snare is such a silly goose

**Comment by Peter0x44** [comment]
I don't own any gc memory cards and I've never used this program, really all I can provide is a few minor wording nitpicks

File: _pages/en_US/gcbackupmanager.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+#### What you need
+
+- An SD card or USB drive
+- [GameCube Backup Manager](https://github.com/AxionDrak/GameCube-Backup-Manager/releases)
+
+To play multiple games, it is recommended to use an external hard drive for your Wii. Any external hard drive found on most modern marketplaces should work with your Wii.
+{: .notice--info}
+
+Make sure your USB drive is formatted as FAT32. Do not format it as other types such as NTFS, exFAT, extFS, or WBFS, the latter being an outdated filesystem for storing Wii games.
```
Comment (Peter0x44) [inline _pages/en_US/gcbackupmanager.md:20]:
```suggestion
Make sure your USB drive is formatted as FAT32. Do not use other filesystems such as NTFS, exFAT, extFS, or WBFS.
```
"types" is a bit vague, so I would change that
Not sure it needs to be explained what WBFS is, but I think this is copied from another page so not your fault
I don't insist on removing that specifically

**Review by Peter0x44 — COMMENTED** [review]
LGTM overall. None of what I brought up is essential, I'll defer this decision to snare
Has my approval.

**Comment by Snare-Hawk** [comment]
eh, i'll just merge it now; the nintendont page will have just have to have more information added to it some other time

**Pull request merged**